[2023-06-23T21:15:41.552+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: running-buses-clean-files-dag.clean-gcp-files scheduled__2023-06-23T21:15:28.471898+00:00 [queued]>
[2023-06-23T21:15:41.559+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: running-buses-clean-files-dag.clean-gcp-files scheduled__2023-06-23T21:15:28.471898+00:00 [queued]>
[2023-06-23T21:15:41.560+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-23T21:15:41.571+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonVirtualenvDecoratedOperator): clean-gcp-files> on 2023-06-23 21:15:28.471898+00:00
[2023-06-23T21:15:41.578+0000] {standard_task_runner.py:57} INFO - Started process 89 to run task
[2023-06-23T21:15:41.584+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'running-buses-clean-files-dag', 'clean-gcp-files', 'scheduled__2023-06-23T21:15:28.471898+00:00', '--job-id', '515', '--raw', '--subdir', 'DAGS_FOLDER/running_buses_clean_files.py', '--cfg-path', '/tmp/tmp2fslaabd']
[2023-06-23T21:15:41.587+0000] {standard_task_runner.py:85} INFO - Job 515: Subtask clean-gcp-files
[2023-06-23T21:15:41.640+0000] {task_command.py:410} INFO - Running <TaskInstance: running-buses-clean-files-dag.clean-gcp-files scheduled__2023-06-23T21:15:28.471898+00:00 [running]> on host 6bba1fead03c
[2023-06-23T21:15:41.703+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='running-buses-clean-files-dag' AIRFLOW_CTX_TASK_ID='clean-gcp-files' AIRFLOW_CTX_EXECUTION_DATE='2023-06-23T21:15:28.471898+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-23T21:15:28.471898+00:00'
[2023-06-23T21:15:41.708+0000] {process_utils.py:181} INFO - Executing cmd: /usr/local/bin/python -m virtualenv /tmp/venv15k5ilaa
[2023-06-23T21:15:41.709+0000] {process_utils.py:185} INFO - Output:
[2023-06-23T21:15:43.074+0000] {process_utils.py:189} INFO - created virtual environment CPython3.11.4.final.0-64 in 708ms
[2023-06-23T21:15:43.076+0000] {process_utils.py:189} INFO -   creator CPython3Posix(dest=/tmp/venv15k5ilaa, clear=False, no_vcs_ignore=False, global=False)
[2023-06-23T21:15:43.079+0000] {process_utils.py:189} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2023-06-23T21:15:43.081+0000] {process_utils.py:189} INFO -     added seed packages: pip==23.1.2, setuptools==67.7.2, wheel==0.40.0
[2023-06-23T21:15:43.085+0000] {process_utils.py:189} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2023-06-23T21:15:43.129+0000] {process_utils.py:181} INFO - Executing cmd: /tmp/venv15k5ilaa/bin/pip install -r /tmp/venv15k5ilaa/requirements.txt
[2023-06-23T21:15:43.133+0000] {process_utils.py:185} INFO - Output:
[2023-06-23T21:15:45.117+0000] {process_utils.py:189} INFO - Collecting google-cloud-storage (from -r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:45.204+0000] {process_utils.py:189} INFO -   Downloading google_cloud_storage-2.9.0-py2.py3-none-any.whl (113 kB)
[2023-06-23T21:15:45.236+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 113.5/113.5 kB 4.4 MB/s eta 0:00:00
[2023-06-23T21:15:45.329+0000] {process_utils.py:189} INFO - Collecting google-auth<3.0dev,>=1.25.0 (from google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:45.354+0000] {process_utils.py:189} INFO -   Downloading google_auth-2.20.0-py2.py3-none-any.whl (181 kB)
[2023-06-23T21:15:45.384+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 181.5/181.5 kB 13.7 MB/s eta 0:00:00
[2023-06-23T21:15:45.462+0000] {process_utils.py:189} INFO - Collecting google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5 (from google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:45.474+0000] {process_utils.py:189} INFO -   Downloading google_api_core-2.11.1-py3-none-any.whl (120 kB)
[2023-06-23T21:15:45.485+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 120.5/120.5 kB 11.6 MB/s eta 0:00:00
[2023-06-23T21:15:45.530+0000] {process_utils.py:189} INFO - Collecting google-cloud-core<3.0dev,>=2.3.0 (from google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:45.553+0000] {process_utils.py:189} INFO -   Downloading google_cloud_core-2.3.2-py2.py3-none-any.whl (29 kB)
[2023-06-23T21:15:45.624+0000] {process_utils.py:189} INFO - Collecting google-resumable-media>=2.3.2 (from google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:45.653+0000] {process_utils.py:189} INFO -   Downloading google_resumable_media-2.5.0-py2.py3-none-any.whl (77 kB)
[2023-06-23T21:15:45.669+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 77.7/77.7 kB 17.4 MB/s eta 0:00:00
[2023-06-23T21:15:45.712+0000] {process_utils.py:189} INFO - Collecting requests<3.0.0dev,>=2.18.0 (from google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:45.733+0000] {process_utils.py:189} INFO -   Downloading requests-2.31.0-py3-none-any.whl (62 kB)
[2023-06-23T21:15:45.739+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.6/62.6 kB 18.7 MB/s eta 0:00:00
[2023-06-23T21:15:45.793+0000] {process_utils.py:189} INFO - Collecting googleapis-common-protos<2.0.dev0,>=1.56.2 (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:45.816+0000] {process_utils.py:189} INFO -   Downloading googleapis_common_protos-1.59.1-py2.py3-none-any.whl (224 kB)
[2023-06-23T21:15:45.827+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 224.5/224.5 kB 23.3 MB/s eta 0:00:00
[2023-06-23T21:15:46.001+0000] {process_utils.py:189} INFO - Collecting protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0.dev0,>=3.19.5 (from google-api-core!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0,<3.0.0dev,>=1.31.5->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:46.017+0000] {process_utils.py:189} INFO -   Downloading protobuf-4.23.3-cp37-abi3-manylinux2014_aarch64.whl (303 kB)
[2023-06-23T21:15:46.032+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 303.5/303.5 kB 22.1 MB/s eta 0:00:00
[2023-06-23T21:15:46.067+0000] {process_utils.py:189} INFO - Collecting cachetools<6.0,>=2.0.0 (from google-auth<3.0dev,>=1.25.0->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:46.084+0000] {process_utils.py:189} INFO -   Downloading cachetools-5.3.1-py3-none-any.whl (9.3 kB)
[2023-06-23T21:15:46.124+0000] {process_utils.py:189} INFO - Collecting pyasn1-modules>=0.2.1 (from google-auth<3.0dev,>=1.25.0->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:46.137+0000] {process_utils.py:189} INFO -   Downloading pyasn1_modules-0.3.0-py2.py3-none-any.whl (181 kB)
[2023-06-23T21:15:46.142+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 181.3/181.3 kB 65.7 MB/s eta 0:00:00
[2023-06-23T21:15:46.177+0000] {process_utils.py:189} INFO - Collecting rsa<5,>=3.1.4 (from google-auth<3.0dev,>=1.25.0->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:46.191+0000] {process_utils.py:189} INFO -   Downloading rsa-4.9-py3-none-any.whl (34 kB)
[2023-06-23T21:15:46.255+0000] {process_utils.py:189} INFO - Collecting six>=1.9.0 (from google-auth<3.0dev,>=1.25.0->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:46.273+0000] {process_utils.py:189} INFO -   Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
[2023-06-23T21:15:46.312+0000] {process_utils.py:189} INFO - Collecting urllib3<2.0 (from google-auth<3.0dev,>=1.25.0->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:46.329+0000] {process_utils.py:189} INFO -   Downloading urllib3-1.26.16-py2.py3-none-any.whl (143 kB)
[2023-06-23T21:15:46.335+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 143.1/143.1 kB 40.7 MB/s eta 0:00:00
[2023-06-23T21:15:46.400+0000] {process_utils.py:189} INFO - Collecting google-crc32c<2.0dev,>=1.0 (from google-resumable-media>=2.3.2->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:46.420+0000] {process_utils.py:189} INFO -   Downloading google_crc32c-1.5.0-cp311-cp311-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (32 kB)
[2023-06-23T21:15:46.500+0000] {process_utils.py:189} INFO - Collecting charset-normalizer<4,>=2 (from requests<3.0.0dev,>=2.18.0->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:46.523+0000] {process_utils.py:189} INFO -   Downloading charset_normalizer-3.1.0-cp311-cp311-manylinux_2_17_aarch64.manylinux2014_aarch64.whl (193 kB)
[2023-06-23T21:15:46.528+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 194.0/194.0 kB 48.5 MB/s eta 0:00:00
[2023-06-23T21:15:46.547+0000] {process_utils.py:189} INFO - Collecting idna<4,>=2.5 (from requests<3.0.0dev,>=2.18.0->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:46.557+0000] {process_utils.py:189} INFO -   Downloading idna-3.4-py3-none-any.whl (61 kB)
[2023-06-23T21:15:46.565+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.5/61.5 kB 22.4 MB/s eta 0:00:00
[2023-06-23T21:15:46.599+0000] {process_utils.py:189} INFO - Collecting certifi>=2017.4.17 (from requests<3.0.0dev,>=2.18.0->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:46.626+0000] {process_utils.py:189} INFO -   Downloading certifi-2023.5.7-py3-none-any.whl (156 kB)
[2023-06-23T21:15:46.645+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 157.0/157.0 kB 28.0 MB/s eta 0:00:00
[2023-06-23T21:15:46.690+0000] {process_utils.py:189} INFO - Collecting pyasn1<0.6.0,>=0.4.6 (from pyasn1-modules>=0.2.1->google-auth<3.0dev,>=1.25.0->google-cloud-storage->-r /tmp/venv15k5ilaa/requirements.txt (line 1))
[2023-06-23T21:15:46.703+0000] {process_utils.py:189} INFO -   Downloading pyasn1-0.5.0-py2.py3-none-any.whl (83 kB)
[2023-06-23T21:15:46.707+0000] {process_utils.py:189} INFO -      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 83.9/83.9 kB 32.2 MB/s eta 0:00:00
[2023-06-23T21:15:46.844+0000] {process_utils.py:189} INFO - Installing collected packages: urllib3, six, pyasn1, protobuf, idna, google-crc32c, charset-normalizer, certifi, cachetools, rsa, requests, pyasn1-modules, googleapis-common-protos, google-resumable-media, google-auth, google-api-core, google-cloud-core, google-cloud-storage
[2023-06-23T21:15:47.511+0000] {process_utils.py:189} INFO - Successfully installed cachetools-5.3.1 certifi-2023.5.7 charset-normalizer-3.1.0 google-api-core-2.11.1 google-auth-2.20.0 google-cloud-core-2.3.2 google-cloud-storage-2.9.0 google-crc32c-1.5.0 google-resumable-media-2.5.0 googleapis-common-protos-1.59.1 idna-3.4 protobuf-4.23.3 pyasn1-0.5.0 pyasn1-modules-0.3.0 requests-2.31.0 rsa-4.9 six-1.16.0 urllib3-1.26.16
[2023-06-23T21:15:47.691+0000] {process_utils.py:181} INFO - Executing cmd: /tmp/venv15k5ilaa/bin/python /tmp/venv15k5ilaa/script.py /tmp/venv15k5ilaa/script.in /tmp/venv15k5ilaa/script.out /tmp/venv15k5ilaa/string_args.txt
[2023-06-23T21:15:47.692+0000] {process_utils.py:185} INFO - Output:
[2023-06-23T21:15:54.098+0000] {process_utils.py:189} INFO - Traceback (most recent call last):
[2023-06-23T21:15:54.100+0000] {process_utils.py:189} INFO -   File "/tmp/venv15k5ilaa/script.py", line 41, in <module>
[2023-06-23T21:15:54.100+0000] {process_utils.py:189} INFO -     res = clean_files(*arg_dict["args"], **arg_dict["kwargs"])
[2023-06-23T21:15:54.101+0000] {process_utils.py:189} INFO -           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2023-06-23T21:15:54.102+0000] {process_utils.py:189} INFO -   File "/tmp/venv15k5ilaa/script.py", line 31, in clean_files
[2023-06-23T21:15:54.103+0000] {process_utils.py:189} INFO -     storage_client = storage.Client()
[2023-06-23T21:15:54.104+0000] {process_utils.py:189} INFO -                      ^^^^^^^^^^^^^^^^
[2023-06-23T21:15:54.105+0000] {process_utils.py:189} INFO -   File "/tmp/venv15k5ilaa/lib/python3.11/site-packages/google/cloud/storage/client.py", line 173, in __init__
[2023-06-23T21:15:54.105+0000] {process_utils.py:189} INFO -     super(Client, self).__init__(
[2023-06-23T21:15:54.106+0000] {process_utils.py:189} INFO -   File "/tmp/venv15k5ilaa/lib/python3.11/site-packages/google/cloud/client/__init__.py", line 320, in __init__
[2023-06-23T21:15:54.107+0000] {process_utils.py:189} INFO -     _ClientProjectMixin.__init__(self, project=project, credentials=credentials)
[2023-06-23T21:15:54.107+0000] {process_utils.py:189} INFO -   File "/tmp/venv15k5ilaa/lib/python3.11/site-packages/google/cloud/client/__init__.py", line 268, in __init__
[2023-06-23T21:15:54.108+0000] {process_utils.py:189} INFO -     project = self._determine_default(project)
[2023-06-23T21:15:54.109+0000] {process_utils.py:189} INFO -               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2023-06-23T21:15:54.110+0000] {process_utils.py:189} INFO -   File "/tmp/venv15k5ilaa/lib/python3.11/site-packages/google/cloud/client/__init__.py", line 287, in _determine_default
[2023-06-23T21:15:54.110+0000] {process_utils.py:189} INFO -     return _determine_default_project(project)
[2023-06-23T21:15:54.111+0000] {process_utils.py:189} INFO -            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2023-06-23T21:15:54.112+0000] {process_utils.py:189} INFO -   File "/tmp/venv15k5ilaa/lib/python3.11/site-packages/google/cloud/_helpers/__init__.py", line 152, in _determine_default_project
[2023-06-23T21:15:54.116+0000] {process_utils.py:189} INFO -     _, project = google.auth.default()
[2023-06-23T21:15:54.117+0000] {process_utils.py:189} INFO -                  ^^^^^^^^^^^^^^^^^^^^^
[2023-06-23T21:15:54.118+0000] {process_utils.py:189} INFO -   File "/tmp/venv15k5ilaa/lib/python3.11/site-packages/google/auth/_default.py", line 692, in default
[2023-06-23T21:15:54.119+0000] {process_utils.py:189} INFO -     raise exceptions.DefaultCredentialsError(_CLOUD_SDK_MISSING_CREDENTIALS)
[2023-06-23T21:15:54.119+0000] {process_utils.py:189} INFO - google.auth.exceptions.DefaultCredentialsError: Your default credentials were not found. To set up Application Default Credentials, see https://cloud.google.com/docs/authentication/external/set-up-adc for more information.
[2023-06-23T21:15:54.180+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/decorators/base.py", line 220, in execute
    return_value = super().execute(context)
                   ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 374, in execute
    return super().execute(context=serializable_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 578, in execute_callable
    result = self._execute_python_callable_in_subprocess(python_path, tmp_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/python.py", line 428, in _execute_python_callable_in_subprocess
    execute_in_subprocess(
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/process_utils.py", line 170, in execute_in_subprocess
    execute_in_subprocess_with_kwargs(cmd, cwd=cwd)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/utils/process_utils.py", line 193, in execute_in_subprocess_with_kwargs
    raise subprocess.CalledProcessError(exit_code, cmd)
subprocess.CalledProcessError: Command '['/tmp/venv15k5ilaa/bin/python', '/tmp/venv15k5ilaa/script.py', '/tmp/venv15k5ilaa/script.in', '/tmp/venv15k5ilaa/script.out', '/tmp/venv15k5ilaa/string_args.txt']' returned non-zero exit status 1.
[2023-06-23T21:15:54.207+0000] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=running-buses-clean-files-dag, task_id=clean-gcp-files, execution_date=20230623T211528, start_date=20230623T211541, end_date=20230623T211554
[2023-06-23T21:15:54.224+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 515 for task clean-gcp-files (Command '['/tmp/venv15k5ilaa/bin/python', '/tmp/venv15k5ilaa/script.py', '/tmp/venv15k5ilaa/script.in', '/tmp/venv15k5ilaa/script.out', '/tmp/venv15k5ilaa/string_args.txt']' returned non-zero exit status 1.; 89)
[2023-06-23T21:15:54.277+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-23T21:15:54.292+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
